Docker/bootstrap.ps1

```diff
@@ -9,33 +9,34 @@
 # and secret with the "Ed-Fi Sandbox" claimset.
 
 param(
-    [string]
-    [ValidateSet("minimal", "populated")]
-    $Template = "minimal"
+    [switch]
+    $Hub
 )
 
+$ErrorActionPreference = "Stop"
+
 if (-not (Test-Path ./gateway/ssl/server.crt)) {
     Push-Location gateway
     ./generate-cert.ps1
     Pop-Location
 }
```

Comment:
This code block must be removed since the gateway container is no longer included.